### DISCUSSION TITLE
Compatibility issues with new UWP software (duplicate, see issue #19)

### DISCUSSION THREAD

**Discussion by lmou523** [sandboxie-plus/Sandboxie#3028] [General]
With Windows 11, a lot of software becomes uwp,sandboxie wasn't very supportive of them.
For example, I open the Explorer in the sandbox, right-click will lose options, double-click a txt file can not open.
![image](https://github.com/sandboxie-plus/Sandboxie/assets/59944420/0c47ce6e-abfc-4891-b5e0-b640152ec4bb)
![image](https://github.com/sandboxie-plus/Sandboxie/assets/59944420/b4e733e7-9276-43fc-8ea0-04b0622d1cc3)

I did a quick search for the reason，When I use Template=OpenCOM，Things will improve。
Apparently this is caused by dcomlaunch, but I don't know exactly why (is communication between dcom and some external service being organized?).

**Comment by isaak654** [comment]
About UWP support, we have an open issue at #19 
Any pull request to provide initial UWP compatibility would definitely be helpful.
